lib/middleware/normalize.ex

```diff
@@ -0,0 +1,19 @@
+defmodule PINXS.Middleware.Normalize do
+  @behaviour Tesla.Middleware
+  alias Tesla.Env
```

Comment:
Might be worth a comment about what this middleware is doing and why.

Reply:
I'll cherry pick the commit I did for V3 as I did add it there